README.md

```diff
@@ -124,7 +124,7 @@ The resulting JavaScript array is:
 ]
 ```
 
-The returned array also exposes a `columns` property containing the column names in input order (in contrast to Object.keys, whose iteration order is arbitrary). For example:
+The returned array also exposes a `columns` property containing the column names, possibly deduplicated, in input order (in contrast to Object.keys, whose iteration order is arbitrary). For example:
```

Comment:
this is super cool! Maybe add some language in here about what will happen when columns are duplicated, just so that there are no surprises?

Reply:
I think this should left to examples, to keep the documentation lean?